scripts/jenkins/fpga/nlb-deploy.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+#
+# Copyright 2019 Intel Corporation.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Deploys current jenkins-build test image for nlb-demo
+# in the cluster.
+
+set -o pipefail
+set -o xtrace
+set -o errexit
+
+REPO_ROOT=$(realpath $(dirname $0)/../../..)
+kubectl create -f ${REPO_ROOT}/demo/test-fpga-region.yaml
```

Comment:
Are there any plans to test FPGA in AF mode?

Reply:
Yes. Eventually, aocx testing, as well. More testing is needed on the backend K8s nodes